packages/ia-components/sandbox/pagination/pagination-with-flexbox.jsx

```diff
@@ -239,7 +239,12 @@ class Paginator extends Component {
     if (numberOfPages === pageSelected) return null;
 
     return (
-      <button type="button" className="pagination-arrow right" onClick={this.goToNextPage}>
+      <button
+        type="button"
+        className="pagination-arrow right"
+        onClick={this.goToNextPage}
+        data-event-click-tracking="Paginator|Arrow-Right"
```

Comment:
Does this need an aria label for accessibility?

Reply:
@koderjoker would like your input on this block of code

Reply:
I agree that the next, previous and page number buttons need aria labels since otherwise they would just read out as "button" on screen readers. We could give our next and previous buttons aria-labels "Go to next page" and "Go to previous page" respectively, and give the page buttons the aria-label={`Go to page number ${isPage ? 'selected' : ''}`}

Reply:
Additionally, (if we want to also tell a user what page they are on) we could change the above to aria-label={isPage ? `Current page, page ${thisPage}` : `Go to page ${thisPage}`}

Reply:
Though I would say that it is mainly the page buttons that require an aria-label, since the next and previous button hold a somewhat reasonable description.

Reply:
@iisa: Separate comment re: click tracking. I know that this Paginator component is generic, and so the click tracking sticks with a generic Category (the first value), but we might want to think about this a little bit. If we have multiple instances of the Paginator, then it's going to be difficult to know which Paginator people clicked on. For the current use case, we want to know how many times the *Music Links* Paginator was clicked. Not sure if there's a good way to pass in the context into the Category value and then use the Action value for the specific affordance, e.g. "Paginator-Arrow-Right". This comment applies to all the click tracking.

Reply:
@bfalling good catch.